pkg/runtime/runtime_test.go

```diff
@@ -3664,6 +3664,7 @@ func NewTestDaprRuntimeWithProtocol(mode modes.DaprMode, protocol string, appPor
 func NewTestDaprRuntimeConfig(mode modes.DaprMode, protocol string, appPort int) *Config {
 	return NewRuntimeConfig(NewRuntimeConfigOpts{
 		ID:                           TestRuntimeConfigID,
+		Namespace:                    os.Getenv("NAMESPACE"),
```

Comment:
Since this is used for tests, better to define a const string.

Reply:
Because uts need to test whether the writing and reading of namespace environment variables are as expected. If this is changed to the default value or fake value, there will be dependencies elsewhere, resulting in failure.

![image](https://user-images.githubusercontent.com/20457624/188769524-e103337b-db4c-425e-8d86-91ad409d620a.png)